gem/templates/springster/patterns/basics/social-media/social-media_with-label.html

```diff
@@ -1,22 +1,20 @@
 {% load wagtailcore_tags wagtailimages_tags wagtailsettings_tags static i18n %}
 {% get_settings %}
 <ul class="social-media-list">
-  <li class="social-media-list__item"><label class="social-media__label">{% trans "Share" %}:</label></li>
+  <li class="social-media-list__item"><label class="social-media__label">{% trans "Share" %}</label></li>
   <!--<li class="social-media-list__item">
```

Comment:
@Mitso I'm overriding the image upload form the CMS, seems pointless on all these sites to load an external images, haven't heard any else regarding instagram tho, surely we'll update this again after launch